### DISCUSSION TITLE
Dable推出新服務YouText網站：以AI將Youtube影片轉換為文章

### DISCUSSION THREAD

**Discussion by Kotori0501** [1101-GS4719-C/Python-Community-GS4719-C#829] [General]
YouTube 影片轉文章，這個技術感覺用在閱讀英文或其它語言身上，感覺會學到很多，當遇到不會的單字時也可以進行翻譯。

https://tw.news.yahoo.com/dable%E6%8E%A8%E5%87%BA%E6%96%B0%E6%9C%8D%E5%8B%99youtext%E7%B6%B2%E7%AB%99-%E4%BB%A5ai%E5%B0%87youtube%E5%BD%B1%E7%89%87%E8%BD%89%E6%8F%9B%E7%82%BA%E6%96%87%E7%AB%A0-010000294.html

**Comment by yuxinlin93** [comment]
如果這個技術拿來實際應用在生活上，遇到影音教材的時候就不怕找不到某格內容的時間點了，也可以輕鬆地把教授的授課內容轉成文字檔，期待期待

**Comment by dabbie109303501** [comment]
這個科技幫助很多出門不戴耳機卻需要找影片內容的人，也讓在查找影片資料時更方便

**Comment by Jasper525** [comment]
如果他轉出來的文字準確度能比YT內建cc字幕高，在搜尋上真的會非常方便。

**Comment by Chiway1229** [comment]
如果有這項技術，我就能在等公車、查資料時看影片更方便了，好耶

**Comment by StevenWanglolz** [comment]
感覺很方便ㄟ，如果沒有耳機的情況下也可以把影片當作文章來讀，而且也可以作筆記!

**Comment by Evan0979** [comment]
感覺不錯欸,有些冗長的影片就可以直接轉文字用讀的就好!
